requirements/edx/base.txt

```diff
@@ -6,6 +6,7 @@
 
 beautifulsoup4==4.1.3
 beautifulsoup==3.2.1
+before_after==0.1.3
```

Comment:
Ugh, I just looked and you're right that won't work. The paver command for installing python prereqs doesn't include development.txt.

OK nvm then, carry on.